### ISSUE TITLE
Latest bw-linux 2024.6.1 isnt compatible with vaultwarden anymore

### ISSUE THREAD

**Issue by peterge-misoft** [dani-garcia/vaultwarden#4720]
### Subject of the issue
We use a CI pipeline that downloads and encrypts a dump of our whole vault. Its running every week. Last week with 2024.6.0 it did work just fine, but this week with 2024.6.1 it says `Invalid master password.` when executing bw unlock. I did not change the pw of course.

### Deployment environment

* vaultwarden version:
Vaultwarden Web Version 2024.1.2 (vaultwarden/server:latest)

* Install method:
docker compose

* Clients used: 
bw-linux 2024.6.1
https://vault.bitwarden.com/download/?app=cli&platform=linux

* Other relevant details:

### Steps to reproduce
```
  script:
    # Install prerequisites
    - apt-get update >/dev/null && apt install unzip sshpass wget -y >/dev/null 
    - wget --no-check-certificate 'https://vault.bitwarden.com/download/?app=cli&platform=linux' -O bw-linux.zip > /dev/null 2>&1
    - unzip bw-linux.zip
    - rm bw-linux.zip
    - chmod +x bw
    # Set timezone to Eurpoe/Berlin for printing date
    - ln -fs /usr/share/zoneinfo/Europe/Berlin /etc/localtime
    - dpkg-reconfigure -f noninteractive tzdata
    # Install certificate
    - echo "$CERTIFICATE" > /root/certificate.pem
    # Set env vars
    - export NODE_EXTRA_CA_CERTS="/root/certificate.pem"
    - export BW_CLIENTID=$BW_CLIENTID
    - export BW_CLIENTSECRET=$BW_CLIENTSECRET
    - export MASTERPASSWORD_EXPORT_USER="$MASTERPASSWORD_EXPORT_USER"
    # Show bw-cli version
    - ./bw -v
    # Configure server
    - ./bw config server https://pass.misoft.local/
    # Login in
    - ./bw login --apikey
    # Export the session key as env var
    - export BW_SESSION=$(./bw unlock --passwordenv MASTERPASSWORD_EXPORT_USER --raw)
    # Export the misoft organization in plaintext
    - ./bw export --organizationid 4ad08680-93e8-400e-a74b-cde5c6e41ce3 --format json --output vaultwarden-misoft-organization-export.json
```

### Expected behaviour
create a dump file, this is the CI output from last week with bw 2024.6.0:
![image](https://github.com/dani-garcia/vaultwarden/assets/162673647/c5738f55-a269-48aa-af2e-9eada6496499)

### Actual behaviour
this is the CI output with bw 2024.6.1:
![image](https://github.com/dani-garcia/vaultwarden/assets/162673647/9639dd5a-58a3-47bd-bb58-c9a53dc91da3)

### Troubleshooting data

see both screenshots

**Comment by otbutz** [comment]
> ### Deployment environment
> 
>     * vaultwarden version:
>       Vaultwarden Web Version 2024.1.2 (vaultwarden/server:latest)

Could you update to vaultwarden [1.31.0](https://github.com/dani-garcia/vaultwarden/releases/tag/1.31.0) ? Your web vault version indicates that you're running vaultwarden [1.30.4](https://github.com/dani-garcia/vaultwarden/releases/tag/1.30.4)

**Comment by peterge-misoft** [comment]
I updated to  Vaultwarden Web Version 2024.5.1, which is 1.31.0.
But the error message is still the same:
![image](https://github.com/dani-garcia/vaultwarden/assets/162673647/6a6ee17c-710a-4667-9270-10592627c653)

**Comment by peterge-misoft** [comment]
When I change the url to use 2024.6.0 like this in the pipeline, it does work again with 1.31.1
![image](https://github.com/dani-garcia/vaultwarden/assets/162673647/6371d39c-e2c8-4fe0-b456-1f8c44129335)
`- wget --no-check-certificate 'https://github.com/bitwarden/clients/releases/download/cli-v2024.6.0/bw-linux-2024.6.0.zip' -O bw-linux.zip > /dev/null 2>&1`

**Comment by BlackDex** [comment]
I just tested it and it and it works fine for me.
I used the exact same arguments (with different org-id of course) and that just outputs the correct json for me.

It only fails if i use the apikey login, then it asks me for the master-password which always reports an invalid master-password.

To note, the sync goes well, after that all is up-to the clients, and nothing is done server-side anymore, for which this is a client issue. I checked the Bitwarden client repo, and found this issue: https://github.com/bitwarden/clients/issues/9919

Going to close this here, as this isn't something we can fix, and Bitwarden already is working on a fix, and it should be resolved in the next cli release.
